docs/developers/contributing.md

```diff
@@ -7,7 +7,12 @@ New contributions to the library are welcome, but we ask that you please follow
 - Check that your code will pass `eslint` code standards, `gulp lint` will run this for you.
 - Check that your code will pass tests, `gulp test` will run tests for you.
 - Keep pull requests concise, and document new functionality in the relevant `.md` file.
-- Consider whether your changes are useful for all users, or if creating a Chart.js plugin would be more appropriate.
+- Consider whether your changes are useful for all users, or if creating a Chart.js [plugin](plugins.md) would be more appropriate.
+
+# Joining the project
+
+ - Active committers and contributors are invited to introduce yourself and request commit access to this project.  Please send an email to a maintainer or file an issue.
```

Comment:
We could merge these two points and remove "send an email..." since most of the time maintainer emails are not public (@users.noreply.github.com). I think the best way to get on board is to join the Slack #dev channel.